app/src/test/java/org/simple/clinic/recentpatient/RecentPatientsScreenControllerTest.kt

```diff
@@ -146,6 +157,15 @@ class RecentPatientsScreenControllerTest {
     ))
   }
 
+  fun `different times to test relative timestamps`() =
+      listOf(
+          listOf(Instant.now(), RelativeTimestamp.Today),
+          listOf(Instant.now().minus(Duration.ofDays(1)), RelativeTimestamp.Yesterday),
+          Instant.now().minus(Duration.ofDays(1)).let { instant ->
```

Comment:
This should be 2 days. It breaks the test right now.

Reply:
oh you're right, fixed. I remember having it as 2, and to test if the test was actually working, I broke it. Forgot to revert it 😅